tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DataStreamsMonitoringManualApiTest.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="DataStreamsMonitoringManualApiTest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.DataStreamsMonitoring;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests;
+
+[UsesVerify]
+public class DataStreamsMonitoringManualApiTest : TestHelper
+{
+    public DataStreamsMonitoringManualApiTest(ITestOutputHelper output)
+        : base("DataStreams.ManualAPI", output)
+    {
+        SetServiceVersion("1.0.0");
+    }
+
+    [SkippableFact]
+    [Trait("Category", "EndToEnd")]
+    public async Task ContextPropagation()
+    {
+        SetEnvironmentVariable(ConfigurationKeys.DataStreamsMonitoring.Enabled, "1");
+
+        using var telemetry = this.ConfigureTelemetry();
+        using var agent = EnvironmentHelper.GetMockAgent(useTelemetry: true);
```

Comment:
You shouldn't set _both_ of these 🙂 And in fact, you're not checking telemetry at all, so I don't think you need _either_ 😄 You _could_ check that the public API metric is being recorded and sent, but tbh, not sure it's worth it 🤷‍♂️